### PR TITLE
Feature/search component testing

### PR DESCRIPTION
**Pull request by AnnaEmson**
### What

Add test for Search component
Add mock data to share across components
Add wrapper component for testing 
Organise and correct some other test components 
correct typo in Collections propTypes 

### How to review

Describe the steps required to test the changes.

### Who can review

Describe who worked on the changes, so that other people can review.

### CODE REVIEW

File: src/app/components/selectable-box/SelectableBoxController.test.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
 import React from "react";
-import SelectableBoxController from "./SelectableBoxController.jsx";
+import SelectableBoxController from "./SelectableBoxController";
 import renderer from "react-test-renderer";
 import { shallow, mount } from "enzyme";
+import JestSnapshot from "../../../node_modules/jest-snapshot/build/index";
```
Comment (dandwal) [inline src/app/components/selectable-box/SelectableBoxController.test.js:5]: More of a question than anything, Does this need to be imported here for anything?
I can see it was using snapshots before without this import.

File: src/app/components/selectable-box/double-column/DoubleSelectableBoxController.test.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import DoubleSelectableBoxController from "./DoubleSelectableBoxController";
+import renderer from "react-test-renderer";
+import { shallow, mount } from "enzyme";
```
Comment (dandwal) [inline src/app/components/selectable-box/double-column/DoubleSelectableBoxController.test.js:4]: Don't think shallow is used in this file, no need to import it

File: src/tests/mockData.js [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+      collectionOwner: "PUBLISHING_SUPPORT",
+      timeseriesImportFiles: [],
+      id: "test-sau39393uyqha8aw8y3n3",
+      name: "Complete collection",
+      type: "manual",
+      teams: ["Team 2"],
+  },
+];
+
+const items = [
```
Comment (dandwal) [inline src/tests/mockData.js:59]: Wonder if we should split this file out a bit maybe things like
- `/tests/mock/collections`
- `/tests/mock/users` 
etc 

I worry people will just add more and more to this file with just slight adjustments then after a while people will just get into the habit of adding things to the file without checking what is there as it becomes rather massive and we could see duplicates.

Also, wonder if we should change this name `items` to something more meaningful, I had to see how it was used to work out what it was for. Might just be `doubleSelectableBoxControllerItems` or something.

**Review by dandwal — COMMENTED** [review]
LGTM minor comments

File: src/app/components/selectable-box/SelectableBoxController.test.js (hunk 1 above)
Comment (AnnaEmson) [reply src/app/components/selectable-box/SelectableBoxController.test.js:5]: oh that is bad.. it was added by auto typing by my VSC, bad boy ... I missed it

File: src/app/components/search/Search.test.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { WrapperComponent } from "../../../tests/test-utils";
+import Search from "./Search.jsx";
+import { mount, shallow } from "enzyme";
+import renderer from "react-test-renderer";
+
+describe("Search", () => {
+    xit("matches the snapshot", () => {
```
[the remark this answers is not in the document]
Comment (AnnaEmson) [reply src/app/components/search/Search.test.js:8]: missed it, sorry

File: src/tests/mockData.js (hunk 3 above)
Comment (AnnaEmson) [reply src/tests/mockData.js:59]: I would leave this fir now as it is. I named the collections as per props name they are requested

**Review by dandwal — APPROVED** [review]

**Pull request merged**